src/commands/actor/charge.ts

```diff
@@ -0,0 +1,81 @@
+import { APIFY_ENV_VARS } from '@apify/consts';
+import { Args, Flags } from '@oclif/core';
+
+import { getApifyTokenFromEnvOrAuthFile } from '../../lib/actor.js';
+import { ApifyCommand } from '../../lib/apify_command.js';
+import { getLoggedClient } from '../../lib/utils.js';
+
+/**
+ * This command can be used to charge for a specific event in the pay-per-event Actor run.
+ * - If run locally or with the --test-pay-per-event flag, it will only log the charge request, without actually charging.
+ * - If run in the Actor run on Apify platform without the testing flag, it will charge the specified amount of events.
+ *
+ * Future TODOs:
+ * - Add logic to work with the max charge USD to prevent exceeding the charging limit.
+ * - Add logic to store events in the log dataset for later inspection to aid local development.
+ */
+export class ChargeCommand extends ApifyCommand<typeof ChargeCommand> {
+	static override description = 'Charge for a specific event in the pay-per-event Actor run.\n';
+
+	static override args = {
+		eventName: Args.string({
+			description: 'Name of the event to charge for',
+			required: true,
+		}),
+	};
+
+	static override flags = {
+		'count': Flags.integer({
+			description: 'Number of events to charge',
+			required: false,
+			default: 1,
+		}),
+		'idempotency-key': Flags.string({
+			description: 'Idempotency key for the charge request',
+			required: false,
+		}),
+		'test-pay-per-event': Flags.boolean({
+			description: 'Test pay-per-event charging without actually charging',
+			required: false,
+			default: false,
+		}),
+	};
+
+	async run() {
+		const { eventName } = this.args;
+		const { count, testPayPerEvent, idempotencyKey } = this.flags;
+
+		const isAtHome = Boolean(process.env.APIFY_IS_AT_HOME);
```

Comment:
in SDK we support `'0'` or `'false'`, both would end up as `true` if you just convert the string to boolean like that. but i guess its not an issue for this particular flag, i cant think of a use case where people would want to set it to a falsy value themselves, since that is the default.